### PR TITLE
Data 331

### PR DESCRIPTION
**Pull request by Ampersandrew**

### CODE REVIEW

File: data/modern/wizards_of_the_coast/msrd/arcana/basics/arcanabasics_classes_advanced.lst [hunk 1]
```diff
@@ -8,14 +8,14 @@ CLASS:Arcane Arranger	HD:6		TYPE:PC.Advanced	CLASSTYPE:PC	MAXLEVEL:10	SOURCEPAGE
...
 3	BONUS:ABILITYPOOL|Arcane Arranger Feat|1
-4													ABILITY:Special Ability|AUTOMATIC|Shadow Resources
-5													ABILITY:Special Ability|AUTOMATIC|Pack Rat
+4													ABILITY:Special Ability|AUTOMATIC|Arcane Arranger ~ FShadow Resources
+5													ABILITY:Special Ability|AUTOMATIC|Arcane Arranger ~ FPack Rat
 6	BONUS:ABILITYPOOL|Arcane Arranger Feat|1
-7													ABILITY:Special Ability|AUTOMATIC|Expert in Your Field
-8								BONUS:VAR|UpMySleeveLvl|1	ABILITY:Special Ability|AUTOMATIC|Up My Sleeve
+7													ABILITY:Special Ability|AUTOMATIC|Shared ~ Expert in Your Field
+8								BONUS:VAR|UpMySleeveLvl|1	ABILITY:Special Ability|AUTOMATIC|Arcane Arranger ~ FUp My Sleeve
```
Comment (LegacyKing) [inline data/modern/wizards_of_the_coast/msrd/arcana/basics/arcanabasics_classes_advanced.lst]: Key does not match.

File: data/modern/wizards_of_the_coast/msrd/arcana/basics/arcanabasics_classes_advanced.lst [hunk 2]
```diff
@@ -8,14 +8,14 @@ CLASS:Arcane Arranger	HD:6		TYPE:PC.Advanced	CLASSTYPE:PC	MAXLEVEL:10	SOURCEPAGE
...
 CLASS:Arcane Arranger	STARTSKILLPTS:6	CSKILL:Bluff|Computer Use|Decipher Script|Diplomacy|Forgery|Gather Information|Intimidate|Knowledge (Arcane Lore)|Knowledge (Business)|Knowledge (Current Events)|Knowledge (Popular Culture)|Knowledge (Streetwise)|Listen|Profession|Read/Write Language|Repair|Research|Sense Motive|SPEAK LANGUAGE
-1													ABILITY:Special Ability|AUTOMATIC|Word on the Street
-2													ABILITY:Special Ability|AUTOMATIC|False Allegiance
+1													ABILITY:Special Ability|AUTOMATIC|Arcane Arranger ~ Word on the Street
+2													ABILITY:Special Ability|AUTOMATIC|Arcane Arranger ~ False Allegiance
 3	BONUS:ABILITYPOOL|Arcane Arranger Feat|1
-4													ABILITY:Special Ability|AUTOMATIC|Shadow Resources
-5													ABILITY:Special Ability|AUTOMATIC|Pack Rat
+4													ABILITY:Special Ability|AUTOMATIC|Arcane Arranger ~ FShadow Resources
+5													ABILITY:Special Ability|AUTOMATIC|Arcane Arranger ~ FPack Rat
```
Comment (LegacyKing) [inline data/modern/wizards_of_the_coast/msrd/arcana/basics/arcanabasics_classes_advanced.lst]: "~ F" added to these keys.

File: data/modern/wizards_of_the_coast/msrd/basics/modern_basics/modernbasics.pcc [hunk 3]
```diff
@@ -23,6 +23,17 @@ COPYRIGHT:PCGen dataset conversion for the Modern System Reference Document Copy
 
 INFOTEXT:This will load the basic options of the MSRD without FX, Menace or races other than Human
 
+# Fact/Factset & Formula System
+#DATACONTROL:modernbasics__datacontrol.lst
+
+### GM Moved
+SAVE:modernbasics__saves.lst
+STAT:modernbasics__stats.lst
+#ALIGNMENT:modernbasics__alignments.lst
```
Comment (LegacyKing) [inline data/modern/wizards_of_the_coast/msrd/basics/modern_basics/modernbasics.pcc]: Modern doesn't use alignment, don't think this is necessary here.

**Comment by LegacyKing** [comment]
Once those issues are rectified, please let me know and this will be ready to be merged. 

Cheers,

**Comment by Ampersandrew** [comment]
Can you have a look at that, please, Andrew

File: data/modern/wizards_of_the_coast/msrd/arcana/basics/arcanabasics_classes_advanced.lst [hunk 4]
```diff
@@ -108,16 +102,16 @@ CLASS:Shadowjack	HD:6		TYPE:PC.Advanced	CLASSTYPE:PC	MAXLEVEL:10	SOURCEPAGE:Arca
 CLASS:Shadowjack	PRESKILL:3,Computer Use=6,Craft (Electronic)=6,Disable Device=6
 # Class Name	Skill Pts/Lvl	Class Skill
 CLASS:Shadowjack	STARTSKILLPTS:6	CSKILL:Computer Use|Craft (Electronic)|Craft (Mechanical)|Craft (Writing)|Decipher Script|Disable Device|Forgery|Gamble|Gather Information|Intimidate|Knowledge (Arcane Lore)|Knowledge (Current Events)|Knowledge (History)|Knowledge (Popular Culture)|Knowledge (Technology)|Profession|Read/Write Language|Repair|Research|SPEAK LANGUAGE|Search
-1														ABILITY:Special Ability|AUTOMATIC|Read/Write Code
-2														ABILITY:Special Ability|AUTOMATIC|Online Presence
+1														ABILITY:Special Ability|AUTOMATIC|Shadowjack ~ Read Write Code
```
Comment (LegacyKing) [inline data/modern/wizards_of_the_coast/msrd/arcana/basics/arcanabasics_classes_advanced.lst]: Why remove "/"? That is a legal character in a key

**Comment by LegacyKing** [comment]
I'll wait for the answer about the key name. But this looks good to me on the whole.

**Comment by Ampersandrew** [comment]
I removed it because I don't like putting maths characters in key names.

Probably not relevant any more since we're going to be removing the legacy
maths parser.

I'm not really attached to having it in or out, but I have a large change
that I've committed locally to
clean up the warnings in the modern set.  I know we usually do that with a
script just before we release,
but I'm overhauling this and want the warnings to be actual warnings.

If you want me to fix this then I can, but it's going to be a pain since
I'm going to have to revert the
change, tweak this and then put it back.  Git is perfectly capable of this
of course.

Let me know what you want.

andrew

**Comment by LegacyKing** [comment]
I like the KEY name to match the original name as much as possible, when we can of course.

Things like "Blind-fight" or "Holy/Unholy" etc, should be maintained. When you can, let's fix that minor deal. However, I understand the "huge" changes, and how fun rebasing can be. Just get it restored when possible. Thanks.

**Comment by Ampersandrew** [comment]
Ok, I'll put it into the branch now.

andrew

**Pull request merged**